src/data/drinkRecipes/spicedAppleFizz.ts

```diff
@@ -0,0 +1,71 @@
+import type { DrinkRecipe } from "types/DrinkRecipe"
+import { DrinkRecipeIngredientType } from "types/DrinkRecipe"
+
+const fakeImageUrl =
+  "https://www.liquor.com/thmb/0Q9zUdEgdxqDdU9aqwhVbmcsz0Q=/960x0/filters:no_upscale():max_bytes(150000):strip_icc()/spiced-apple-fizz-720x720-primary-0bbfd940b7fb473284344b2ae591864a.jpg"
+
+const spicedAppleFizz: DrinkRecipe = {
+  name: "Spiced Apple Fizz",
+  imageUrl: fakeImageUrl,
+  measuredIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Vodka",
+      unit: "oz",
+      unitAmount: 1.5,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Pear Brandy",
+      unit: "oz",
+      unitAmount: 0.5,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Lemon Juice",
+      unit: "oz",
+      unitAmount: 0.5,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Honey Syrup",
+      unit: "oz",
+      unitAmount: 0.5,
+      alternates: [],
+    },
+  ],
+  numberedIngredients: [],
+  portionlessIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Portionless,
+      ingredient: "Hard Apple Cider",
+      alternates: [],
+    },
+  ],
+  garnishIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Portionless,
+      ingredient: "Apple",
+      suffix: "Fan",
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Portionless,
+      prefix: "Grated",
+      ingredient: "Cinnamon",
+      alternates: [],
+    },
+  ],
+  stepsToMake: [
+    "Add %MI1, %MI2, %MI3 and %MI4 to cocktail shaker filled with ice.",
+    "Shake until well-chilled",
```

Comment:
So something that I don't want you to address here - I've been meaning to - is this usage of sometimes ending steps with periods and other times not. I think I'm going to add a test to assert that steps don't have periods in them. That's totally separate from your PR here and I don't want you to change anything in that regard for now, but my thinking is that if we prevent periods from being in the steps, then you can't have a single step be multiple sentences long (because then it's several steps just crammed into one and that bothers me; even single sentence steps sometimes just concatenate two steps into one with a conjunction) and also that allows us to programmatically decide later whether / how we show them for each step.